Code/Framework/AzCore/AzCore/Debug/Budget.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "Budget.h"
+
+#include <AzCore/Module/Environment.h>
+#include <AzCore/Math/Crc.h>
+#include <AzCore/Memory/SystemAllocator.h>
+
+AZ_DEFINE_BUDGET(AzCore);
+AZ_DEFINE_BUDGET(Editor);
+AZ_DEFINE_BUDGET(Entity);
+AZ_DEFINE_BUDGET(Game);
```

Comment:
Should this one be moved the AzGameFramework? If not then perhaps a more generic name such as "Simulation" could be used.

Reply:
I don't know actually what the distinction between `AzGameFramework` and `Game` is actually as used throughout the engine, but I have this marked as a future todo task (along with cleaning up a bunch of other marker budgets I saw along the way but opted not to fix in this pass directly)